recipes/arbiter/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir, save
+
+required_conan_version = ">=1.53.0"
+
+
+class PackageConan(ConanFile):
+    name = "arbiter"
+    description = "Uniform access to the filesystem, HTTP, S3, GCS, Dropbox, etc."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/connormanning/arbiter"
+    topics = ("filesystem", "http", "s3", "gcs", "dropbox")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libcurl/[>=7.78.0 <9]")
+        self.requires("openssl/[>=1.1 <4]")
+        self.requires("zlib/[>=1.2.11 <2]")
+        self.requires("nlohmann_json/3.11.2", transitive_headers=True)
```

Comment:
Could we upgrade to the below version to prevent dependency mismatch error
```
diff --git a/recipes/arbiter/all/conanfile.py b/recipes/arbiter/all/conanfile.py
index 5045c10f6..eb44c83d5 100644
--- a/recipes/arbiter/all/conanfile.py
+++ b/recipes/arbiter/all/conanfile.py
@@ -49,7 +49,7 @@ class PackageConan(ConanFile):
         self.requires("libcurl/[>=7.78.0 <9]")
         self.requires("openssl/[>=1.1 <4]")
         self.requires("zlib/[>=1.2.11 <2]")
-        self.requires("nlohmann_json/3.11.2", transitive_headers=True)
+        self.requires("nlohmann_json/3.11.3", transitive_headers=True)
         self.requires("rapidxml/1.13", transitive_headers=True)
 
     def validate(self):

```

Reply:
@valgur any updates regarding this?